src/main/java/seedu/address/logic/commands/GithubCommand.java

```diff
@@ -0,0 +1,67 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.awt.Desktop;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.GithubUsername;
+import seedu.address.model.person.Person;
+
+/**
+ * Opens Github User profile page on browser.
+ */
+public class GithubCommand extends Command {
+
+    public static final String COMMAND_WORD = "github";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Opens up Github Profile page associated to chosen "
```

Comment:
I believe you can be more specific by specifying who the Github Profile page associated to.
```suggestion
    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Opens up Github Profile page associated to index specified "
```